deploy/osd-limited-support/policy-generator-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+    name: osd-limited-support
+policyDefaults:
+    namespace: openshift-osd-metrics
+    placement:
+      clusterSelectors:
+        api.openshift.com/limited-support: true
+    remediationAction: enforce
+policies:
+  - name: osd-limited-support-configmap
+    manifests:
+      - path: 00-limited-support.ConfigMap.yaml
```

Comment:
Do you still need this file? Because the `generate-policy-config.py` script would fill this out and save it with the other configs in the temp_directory. 

If the file is still needed, i think it should be moved to another directory, that's not /deploy (maybe /script or /resources). that's because this policy-generator-config.yaml is here, it will be included in the selectorsyncset.

Reply:
Good catch, this does need to be removed.